### PR TITLE
Correct spelling errors and include omitted words in the documentation.

### PR DESCRIPTION
**Pull request by Vivek-04022001**
**I'm reading the Volto Hands-On documentation, and I've encountered some mistakes. I've raised a pull request to correct the errors.**

I, Vivek Kumar, agree to have this contribution published under the Creative Commons 4.0 International License (CC BY 4.0), with attribution to the Plone Foundation.

### CODE REVIEW

**Comment by stevepiercy** [comment]
@Vivek-04022001 I noticed you inserted an extra word "the" that must be removed from the agreement statement in your description.

> under _the_ Creative Commons 4.0 International License

Sorry to be nitpicky. In the future, we suggest that you sign the [Plone Contributor Agreement](https://plone.org/foundation/contributors-agreement) so you don't have to do this for every pull request.

File: docs/voltohandson/quickstart.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To create our first volto project we will install the volto generator tool (http
 ```shell
 yo @plone/volto
 ```
-The dialogue in the console will ask you first to prompt a name for your project. When choosing an apropriate name, please note that due to a current bug your name should not begin with a number. After you have given a name to your project, you will be a wether you want to install any addons. You can skip by pressing the `enter` button, as we dont want to use any 3rd party addons for the training.
+The dialogue in the console will ask you first to prompt a name for your project. When choosing an appropriate name, please note that due to a current bug your name should not begin with a number. After you have given a name to your project, you will be asked weather you want to install any addons. You can skip by pressing the `enter` button, as we don't want to use any third-party addons for the training.
```
Comment (stevepiercy) [inline docs/voltohandson/quickstart.md:21]: This has additional English grammar, MyST syntax enhancements, and conforms to the [Plone documentation styleguide](https://6.docs.plone.org/contributing/documentation/authors.html#plone-documentation-styleguide) of one sentence per line in narrative docs.

```suggestion
The dialogue in the console will first prompt you for a name for your project.
When choosing an appropriate name, please note that due to a current bug, your name should not begin with a number.
After you have given a name to your project, you will be asked whether you want to install any add-ons.
You can skip ahead by pressing the {kbd}`enter` button, as we don't want to use any third-party add-ons for the training.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by Vivek-04022001** [comment]
Apologies, @stevepiercy. I'm familiar with MyST syntax because you provided details when I submitted my first pull request. This time, it was an accidental oversight on my part. I'll ensure not to write an entire paragraph in just one line in the future.

I've already signed the [Plone Agreement](https://plone.org/foundation/contributors-agreement), but sometimes, I worry that my pull requests might not be accepted if I don't explicitly mention it in the description. However, moving forward, I won't include it with every pull request. Thank you for your feedback.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@Vivek-04022001 thanks for the revision and update. Let's wait until you get that email notification, and then when you get it, would you please add a comment and `@` to merge this PR? It looks good, but that agreement piece is required. Thank you!

**Comment by Vivek-04022001** [comment]
@stevepiercy, I'm a bit confused about the instruction to "add a comment and @ to merge this PR." I have already received an email from Plone, which includes the PDF with all the necessary agreements mentioned.
[PLONE FOUNDATION contributor agreement Plone.pdf](https://github.com/plone/training/files/14052812/PLONE.FOUNDATION.contributor.agreement.Plone.pdf).

**Comment by stevepiercy** [comment]
@Vivek-04022001 I don't see an email address on your agreement. From the last FAQ on https://plone.org/foundation/contributors-agreement:

> ### I signed the Agreement, but haven't been given access yet?
> Please allow a week for a response. Volunteers process the Contributor Agreements. If you have issues, such as making a spelling error in your email or GitHub username, please contact [agreements@plone.org](mailto:agreements@plone.org).

I would suggest that you follow up to see what might have gone wrong.

**Comment by Vivek-04022001** [comment]
Screenshot 2024-01-26 015040
Is it necessary for me to compose an email and include my Gmail address in this agreement, even though my GitHub username is provided?

**Comment by stevepiercy** [comment]
One more time, this time with _feeling_.

> please contact [agreements@plone.org](mailto:agreements@plone.org).

**Comment by Vivek-04022001** [comment]
@stevepiercy, could you please verify my name? I received an email stating that I've been invited now.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@Vivek-04022001 something is not right. Please make sure your emails on GitHub match what you put on the Agreement. If there is something else that does not align, then please contact agreements@plone.org.

**Comment by Vivek-04022001** [comment]
Hello @stevepiercy, sorry to bother you, but could you please double-check my name? I just received another email confirming my invitation.

**Comment by stevepiercy** [comment]
@Vivek-04022001 thanks for the notice. It looks like it was corrected. Thank you for your patience and follow through. Merging.

**Pull request merged**